src/index.js

```diff
@@ -0,0 +1,86 @@
+// import packages
+const inquirer = require("inquirer");
+const questions = require("./utils/questions");
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const generator = require("./utils/generator");
+const fs = require("fs");
+const path = require("path");
+
+// declare global variables
+const engineers = [];
+const interns = [];
+
+const init = async () => {
+  // declare variables
+  let inProgress = true;
+
+  // ask initial questions
+  const managerInfo = await inquirer.prompt(questions.getManagerQuestions());
+
+  setManager(managerInfo);
+
+  let continueChoice = await inquirer.prompt(questions.getContinueQuestion());
+
+  // loop questions until full team has been created
+  while (inProgress) {
+    if (continueChoice.continueOption == "Add a new engineer") {
+      const engineerAnswers = await inquirer.prompt(
+        questions.getEngineerQuestions()
+      );
+      addEngineer(engineerAnswers);
+      continueChoice = await inquirer.prompt(questions.getContinueQuestion());
+    } else if (continueChoice.continueOption == "Add a new intern") {
+      const internAnswers = await inquirer.prompt(
+        questions.getInternQuestions()
+      );
+      addIntern(internAnswers);
+      continueChoice = await inquirer.prompt(questions.getContinueQuestion());
+    } else {
+      inProgress = false;
+    }
+  }
+  const html = generator.generateHTML(engineers, interns);
+
+  fs.writeFileSync(path.join(__dirname, "../dist", "./teamProfile.html"), html);
+
+  console.log("Successfully Generated HTML");
+};
+
+// create manager
+const setManager = (managerInfo) => {
```

Comment:
Destructure the parameters here:

`const setManager = ( { managerName, managerId, managerEmail, officeNumber, teamName } ) => { ... `